Makefile

```diff
@@ -50,30 +50,40 @@ clean-wasi-sdk:
 test: compile-examples
 		@extism call examples/simple_js.wasm greet --wasi --input="Benjamin"
 		@extism call examples/bundled.wasm greet --wasi --input="Benjamin" --allow-host "example.com"
-ifeq ($(OS),Windows_NT)
-		@python3 -m venv ./.venv && \
-			./.venv/Scripts/activate.bat && \
-			pip install -r examples/host_funcs/requirements.txt && \
-			python examples/host_funcs/host.py examples/host_funcs.wasm && \
-			./.venv/Scripts/deactivate.bat
-else
-		@python3 -m venv ./.venv && \
-			. ./.venv/bin/activate && \
-			pip install -r examples/host_funcs/requirements.txt && \
-			python3 examples/host_funcs/host.py examples/host_funcs.wasm && \
-			deactivate
-endif
+# ifeq ($(OS),Windows_NT)
+# 		@python3 -m venv ./.venv && \
+# 			./.venv/Scripts/activate.bat && \
+# 			pip install -r examples/host_funcs/requirements.txt && \
+# 			python examples/host_funcs/host.py examples/host_funcs.wasm && \
+# 			./.venv/Scripts/deactivate.bat
+# else
+# 		@python3 -m venv ./.venv && \
+# 			. ./.venv/bin/activate && \
+# 			pip install -r examples/host_funcs/requirements.txt && \
+# 			python3 examples/host_funcs/host.py examples/host_funcs.wasm && \
+# 			deactivate
+# endif
```

Comment:
I'm gonna leave this uncommented and merge.

Reply:
@chrisdickinson no rush, but could you chime in as to how to fix this or when another sdk release will happen? i don't recall we had a definitive reason why we were still getting the failure to import here.

Reply:
I wasn't able to replicate the `extism_sys` issue on my machine – I ran into a _similar_ issue with `extism-cli` but I solved it by upgrading my virtualenv. I think it _could_ be a virtualenv issue – looking back at the error you posted, it doesn't look like you had the target virtualenv active when you uninstalled `extism_sys` and installed a newer version. The CI looks like it was failing for [a different reason](https://github.com/extism/js-pdk/actions/runs/12164760947/job/33927191400?pr=113#step:9:880), though?

Reply:
This must have been some kind of caching issue, the tests are passing now: https://github.com/extism/js-pdk/pull/116